content/docs/started/getting-started.md

```diff
@@ -78,6 +78,14 @@ Download, set up, and deploy (If you prefer to work from source code, feel free
 **Important**: The commands above will enable collection of **anonymous** user data to help us improve Kubeflow; for more information including instructions for explicitly
 disabling it please refer to the [usage reporting guide](/docs/other-guides/usage-reporting/).
 
+## Remove KubeFlow
+To remove KubeFlow use the same script as above. Note, that it will delete the namespace kubeflow along with everything you have deployed in it!
```

Comment:
Recommended wording to improve clarity:

```
To remove your Kubeflow deployment, you can use the same `kfkctl` script as above.
```

Reply:
Sure sounds nice as well. Lets go with that. How can I change it?